_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,132 @@
+for more details.
+
+
+Flink 1.8.0 is API-compatible with previous 1.x.y releases for APIs annotated
+with the `@Public` annotation.  The release is available now and we encourage
+everyone to [download the release](http://flink.apache.org/downloads.html) and
+check out the updated
+[documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.8/).
+Feedback through the Flink [mailing
+lists](http://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/projects/FLINK/summary) is, as always,
+very much appreciated!
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html) on the Flink project site.
+
+{% toc %}
+
+With Flink 1.8.0 we come closer to our goals of enabling fast data processing
+and building data-intensive applications for the Flink community in a seamless
+way. We do this by cleaning up and refactoring Flink under the hood to allow
+more efficient feature development in the future. This includes removal of the
+legacy runtime components that were subsumed in the major rework of Flinks
+underlying distributed system
+([FLIP-6](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077))
+as well as refactorings on the Table API that prepare it for the future
+addition of the Blink enhancements
+([FLINK-11439](https://issues.apache.org/jira/browse/FLINK-11439)).
+
+Nevertheless, this release includes some important new features and bug fixes.
+The most interesting of those we'll highlight below. Please consult the
+[complete changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344274)
+or the [release notes](https://ci.apache.org/projects/flink/flink-docs-release-1.8/release-notes/flink-1.8.html)
+for more details.
+
+
+## New Features and Improvements
+
+* **New Support for Schema Migration when restoring Savepoints**: With Flink
+  1.7.0 we added support for changing the schema of state when using the
+  `AvroSerializer`
+  ([FLINK-10605](https://issues.apache.org/jira/browse/FLINK-10605)). With
+  Flink 1.8.0 we made great progress migrating all built-in `TypeSerializers`
+  to a new serializer snapshot abstraction that theoretically allows schema
+  migration. Of the serializers that come with Flink, we now support schema
+  migration for the `PojoSerializer`
+  ([FLINK-11485](https://issues.apache.org/jira/browse/FLINK-11485)) and Java
+  `EnumSerializer`
+  ([FLINK-11334](https://issues.apache.org/jira/browse/FLINK-11334)), as well
+  as for Kryo in limited cases
+  ([FLINK-11323](https://issues.apache.org/jira/browse/FLINK-11323)).
+
+
+* **Continuous cleanup of old state based on TTL
+  ([FLINK-7811](https://issues.apache.org/jira/browse/FLINK-7811))**: We
+  introduced TTL (time-to-live) for Keyed state in Flink 1.6
+  ([FLINK-9510](https://issues.apache.org/jira/browse/FLINK-9510)).  This
+  feature allowed to clean up and make inaccessible keyed state entries when
```

Comment:
"to clean up and make inaccessible keyed state entries when accessing them" - a bit unclear. Do you mean "enabled cleanup and making keyed state entries inaccessible after a defined timeout" ?

Reply:
Taking your suggestions, that sounds quite a bit better. 😅